cpp/src/arrow/flight/types.h

```diff
@@ -554,6 +556,9 @@ class ARROW_FLIGHT_EXPORT FlightInfo {
   /// The total number of bytes in the dataset. If unknown, set to -1
   int64_t total_bytes() const { return data_.total_bytes; }
 
+  /// Whether endpoints are in the same order as the data.
+  bool ordered() const { return data_.ordered; }
```

Comment:
I am not familiar with flight so maybe a dumb question: does the user know which order it is? Like ascending/descending and what are the sorted columns?

Reply:
Good question, but no, this is just a semantic flag to indicate that (unlike the current status) there is _some_ ordering on the dataset. It's still up to the application to define what that means.